### DISCUSSION TITLE
v2.5.1

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#3764] [分享 / Share]
- 修复 `展开选集列表` 和 `展开选集标题` 冲突的问题, 这两个组件将合并在一起, 并提供单独的选项控制列表和标题. (#3752, #692)
> 如果在 v2.5.0 中尝试安装过 `展开选集列表`, 记得先卸载掉.

**⚠ 由于 CDN 缓存, jsDelivr 源的 `在线仓库` 功能需要等到明天 (2022-10-23) 才能恢复使用.**

- 修复 jsDelivr 源中 MDI 图标库加载失败. (#3752, #3761)
- 修复 `简化首页` 在带 query 参数的首页不显示分区选项. (#3730)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.5.1'>v2.5.1</a>.</em>

**Comment by Geeyun-JY3** [comment]
目前 UP 主是可以在合集里进行分组的，而目前 `选集区域优化` 的 `展开选集列表` 会使合集的所有分组都展开且无法折叠收起，是否有必要允许用户自由折叠收起展开合集里的分组？
![bilibili_channel_collection](https://user-images.githubusercontent.com/49396421/198873073-9519cd94-c2e5-449a-b956-20c5227c1827.gif)

例子：https://www.bilibili.com/video/av257302621/

另外破站的默认逻辑好像是只展开当前视频所在分组，其他分组折叠收起。

**Reply by the1812** [reply]
我看看能不能只强制展开打开的分组

**Reply by the1812** [reply]
没找到分组上能够区分的地方...

**Reply by the1812** [reply]
找到个办法了, 只是收起时稍微有点延迟, 问题不大

**Comment by scdyysf** [comment]
目前“下载视频”功能选择格式为“flv”时，“清晰度”只有“1080p”和“360p”两个选项，不能下载其他清晰度

**Reply by Geeyun-JY3** [reply]
见：#3748、#3773
